File: code-coaches/medium/roadrunner/roadrunner.c
#include <stdio.h>

int is_roadrunner_safe(int distance_to_safety, int roadrunner_speed, int coyote_speed)
{
    double seconds_to_safety = distance_to_safety / roadrunner_speed;
    double roadrunner_distance = roadrunner_speed * seconds_to_safety;
    double coyote_distance = coyote_speed * seconds_to_safety - 50;
    return roadrunner_distance > coyote_distance;
}

int main()
{
    int distance_to_safety, roadrunner_speed, coyote_speed;
    scanf("%d%d%d", &distance_to_safety, &roadrunner_speed, &coyote_speed);

    if (is_roadrunner_safe(distance_to_safety, roadrunner_speed, coyote_speed))
        printf("Meep Meep");
    else
        printf("Yum");

    return 0;
}

File: code-coaches/medium/roadrunner/roadrunner.rb
def is_roadrunner_safe(distance_to_safety, roadrunner_speed, coyote_speed)
  seconds_to_safety = distance_to_safety.to_f / roadrunner_speed
  roadrunner_distance = roadrunner_speed * seconds_to_safety
  coyote_distance = coyote_speed * seconds_to_safety - 50
  roadrunner_distance > coyote_distance
end

def main
  distance_to_safety = gets.chomp.to_i
  roadrunner_speed = gets.chomp.to_i
  coyote_speed = gets.chomp.to_i

  if is_roadrunner_safe(distance_to_safety, roadrunner_speed, coyote_speed)
    puts "Meep Meep"
  else
    puts "Yum"
  end
end

if __FILE__ == $0
  main
end